### PR TITLE
Simple tool to remove group members matching a regular expression

### PR DESCRIPTION
**Pull request by lgruen**

### CODE REVIEW

File: groups_cleanup/main.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    while True:
+        search_query = urlencode(
+            {
+                'page_size': PAGE_SIZE,
+                'page_token': next_page_token,
+            }
+        )
+        search_members_request = service.groups().memberships().list(parent=group_name)
+        param = '&' + search_query
+        search_members_request.uri += param
```
Comment (illusional) [inline groups_cleanup/main.py:61]: Wow that's super gross haha

File: groups_cleanup/main.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        response = search_members_request.execute()
+
+        if 'memberships' not in response:
+            break
```
Comment (illusional) [inline groups_cleanup/main.py:65]: Does this happen often, or just defensive?

**Review by illusional — APPROVED** [review]

File: groups_cleanup/main.py (hunk 1 above)
Comment (lgruen) [reply groups_cleanup/main.py:61]: Based on the official documentation...! :-/
https://cloud.google.com/identity/docs/how-to/create-google-groups#searching_for_a_google_group

File: groups_cleanup/main.py (hunk 2 above)
Comment (lgruen) [reply groups_cleanup/main.py:65]: This happens for empty groups.

**Pull request merged**